README.md

```diff
@@ -22,8 +22,10 @@ This plugin can be installed from the [Update Center][3] (found at `Manage Jenki
 
 You can use two ways to configure your plugin to submit data to Datadog:
 
-* Sending the data directly to Datadog through HTTP.
-* Using a DogStatsD server that acts as a forwarder between Jenkins and Datadog.
+* **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
+  - Build Logs collection only works with a full Datadog Agent installed.
+* Sending data directly to Datadog through HTTP.
+  - The HTTP client implementation used is blocking with a timeout duration of 1 minute. If there is a connection problem with Datadog, it will slow your Jenkins instance down.
```

Comment:
Is it a sure thing ? Have you tried ?
```suggestion
  - The HTTP client implementation used is blocking with a timeout duration of 1 minute. If there is a connection problem with Datadog, it may slow your Jenkins instance down.
```

Reply:
yes. I tried and it's bad :\